firmware-binaries/clock-control-reg-cpy/src/main.rs

```diff
@@ -12,7 +12,7 @@ use riscv_rt::entry;
 
 #[cfg_attr(not(test), entry)]
 fn main() -> ! {
-    let mut cc = unsafe { ClockControl::from_base_addr(0xC000_0000 as *const u32) };
+    let mut cc = unsafe { ClockControl::from_base_addr(0xC000_000C as *const u32) };
```

Comment:
How is this still the case after the debug register change?

Reply:
Missed this binary when updating the others. Should be fixed now.